### ISSUE TITLE
Search page filter a11y

### ISSUE THREAD

**Issue by verityroberts** [oaknational/Oak-Web-Application#1704]
Filters on the search page are showing a11y errors incl:
- missing form label on search bar
- multiple form labels on the checkbox

Also alerts on the checkboxes not being in fieldsets - can we please add those for the 3 filter types

Possibly look at how IDs are handled on the input component

**Acceptance Criteria:**
- [ ] No issues when you run pa11y on search
- [ ] Search still works

**Comment by jack-skerrett-oak** [comment]
also seen here: https://github.com/oaknational/Oak-Web-Application/issues/1608

**Comment by jack-skerrett-oak** [comment]
moved to notion
